### ISSUE TITLE
geeup upload metadata not in asset

### ISSUE THREAD

**Issue by tylanderson** [samapriya/geeup#4]
Hi Samapriya,

I have run into issues getting metadata properties to upload into the assets, but I can't pin down where it is happening. No errrors are thrown when uploading the asset, however the properties and metadata does not end up in the asset online. 

Here is the code I am running.
`geeup upload --source E:\Moth\HLS_DATA\HLS_GTiff --dest users/tiller567/testUpload3 -u tiller567@gmail.com -m E:\Moth\HLS_DATA\metadata3.csv`

From this, in the console I get this output (modified the code to print out properties to make sure they were formatted correctly).

```
New collection users/tiller567/testUpload4 created
Processing image 1 out of 237: E:\Moth\HLS_DATA\HLS_GTiff\L2AT19TBG2015334HLS14_MTLstack.tif
{'id_no': 'L2AT19TBG2015334HLS14_MTLstack', 'system:time_start': '2015-11-30T15:51:31.460Z', 'cloud_coverage': 55, 'NCOLS': 3660, 'NROWS': 3660, 'SENSOR': 'Sentinel-2A', 'spatial_coverage': 8, 'SPATIAL_RESOLUTION': 30}
Upload started for: users/tiller567/testUpload4/L2AT19TBG2015334HLS14_MTLstack
Started upload task with ID: WN2IT5DCGXSI3744477C7KCI
Processing image 2 out of 237: E:\Moth\HLS_DATA\HLS_GTiff\L2AT19TBG2015344HLS14_MTLstack.tif
{'id_no': 'L2AT19TBG2015344HLS14_MTLstack', 'system:time_start': '2015-12-10T15:51:31.461Z', 'cloud_coverage': 28, 'NCOLS': 3660, 'NROWS': 3660, 'SENSOR': 'Sentinel-2A', 'spatial_coverage': 11, 'SPATIAL_RESOLUTION': 30}
```

Any idea if there is an issue with my code or if its a server side related issue with google earthengine?

**Comment by samapriya** [comment]
Hi @tylanderson 
This is an interesting problem, it started with Earth Engine started to require explicitly setting the type of metadata property for each metadata field. So for example if the first image being ingested has a satellite id that is completely numeric the metadata property is set to numeric, however, if your next image has a sat ID which is alphanumeric, the ingest will fail with inconsistent property type. So this is something I should have removed as a flag from the tool. Now, this can be avoided moving into the manifest methodology. For now I am converting this into an enhancement request for the tool and hopefully, I will find time soon to work on this.

**Comment by samapriya** [comment]
Hi @tylanderson 
Just released v0.2.2 should now be able to explicitly cast metadata type and upload to earth engine. Closing the issue as feature enhancement is complete.
